Makefile

```diff
@@ -44,6 +44,7 @@ DOCKER_CMD     = docker run --rm -ti -v $(PWD):/go/src/$(SC_PKG) \
 
 ifneq ($(origin DOCKER),undefined)
   # If DOCKER is defined then make it the full docker cmd line we want to use
+  USE_DOCKER=DOCKER=1
```

Comment:
ok- I still need something special even with that change. Since "DOCKER" will either be blank or "1" I need an easy way to add "DOCKER=1" or "" to a sub-cmd. Only way I can think of is to define a "DOCKER_FLAG" var to be "DOCKER=1" or "" based on whether the user turned on Docker, so I can't fully get rid of this line - I just renamed "USE_DOCKER" to "DOCKER_FLAG". Hopefully, that's a bit more meaningful.